qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/jupyterhub-pr.yaml

```diff
@@ -68,3 +68,10 @@ jobs:
           tf_actions_subcommand: 'plan'
           tf_actions_working_dir: 'infrastructure'
           tf_actions_comment: true
+          {% if cookiecutter.prefect is defined %}
+          {% if cookiecutter.prefect %}
```

Comment:
I think ` {% if cookiecutter.prefect %}` is redundant.

Reply:
What if we need to make it false ?

Reply:
Ok, will remove it